armi/physics/fuelCycle/translationFunctions.py

```diff
@@ -0,0 +1,676 @@
+import json
+import math
+import numpy
+from armi import utils
+from armi.reactor.flags import Flags
+
+
+def buildRingSchedule(
```

Comment:
It looks like you still need to complete the refactor of the code in `fuelHandlers` to replace with these functions. Can you verify when this is done?